backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/MetricsApiImpl.scala

```diff
@@ -91,7 +91,6 @@ class MetricsApiImpl extends MetricsApi with Logging {
     Map(
       "rawInputRows" -> SQLMetrics.createMetric(sparkContext, "number of raw input rows"),
       "rawInputBytes" -> SQLMetrics.createSizeMetric(sparkContext, "number of raw input bytes"),
-      "outputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"),
```

Comment:
Should we replace it with the new name instead of deleting?

Reply:
I think for HiveTableScan and FileSourceScan, we defined both "outputRows" and "numOutputRows".

Reply:
There's already numOutputRows at line 115